lib/Elastica/Query/Fuzzy.php

```diff
@@ -57,13 +57,13 @@ public function setFieldOption($param, $value)
     {
         //Retrieve the single existing field for alteration.
         $params = $this->getParams();
-        if (count($params) < 1) {
+        if (empty($params)) {
```

Comment:
I prefer to keep the count(..) here for the "meaning" of the code. Behaviour should be the same. Perhaps even more meaningful would be `count(...) == 0`